modules/ROOT/pages/aggregator/aggregator-examples.adoc

```diff
@@ -23,7 +23,7 @@ So, you could use this:
 </aggregators:size-based-aggregator>
 ----
 
-Now, suppose that you want to execute some action every time your list of names is filled by a 10%. You do not want to do this for every new element because that would cause too many executions. You can assume that the logic to be executed is in another flow named `partialAction`. We can you the `incremental-aggregation` route for this.
+If you want to execute some action every time your list of names is filled by a 10%. You do not want to do this for every new element because that would cause too many executions. You can assume that the logic to be executed is in another flow named `partialAction`. We can you the `incremental-aggregation` route for this.
```

Comment:
The first sentence is not grammatical. I don't know if you want to fix it in this PR.